doc/template.html

```diff
@@ -25,7 +25,7 @@
       <header>
         <div class="header-container">
           <h1>Node.js __VERSION__ Documentation</h1>
-          <button class="theme-toggle-btn" id="theme-toggle-btn" hidden>
+          <button class="theme-toggle-btn" id="theme-toggle-btn" aria-label="Toggle dark mode/light mode" hidden>
```

Comment:
```suggestion
          <button class="theme-toggle-btn" id="theme-toggle-btn" title="Toggle dark mode/light mode" hidden>
```

No strong opinion, but `title` also helps those who hover the element with their cursor.

Reply:
Using `title` attribute for a tooltip is [generally undesirable from an accessibility point of view](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#accessibility_concerns). It's possible that a lot (most? all?) of the relevant concerns may be addressed in this particular situation by also having the `aria-label` attribute. But I'd want to look at it more closely. I'll add a tooltip in a follow-on PR (unless someone beats me to it).